pnpm-workspace.yaml

```diff
@@ -1,3 +1,4 @@
 packages:
   - "examples/*"
   - "packages/*"
+  - "!docs"
```

Comment:
Yeah, you're right that this doesn't do anything meaningful, since `docs/` was already not included in the prior patterns.

Reply:
Should I remove it from the list or leave it there to be explicit?

Reply:
Totally up to you!